litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/ChaosTable/TableData.tsx

```diff
@@ -25,46 +27,39 @@ const TableData: React.FC<TableDataProps> = ({
   return (
     <>
       <StyledTableCell padding="checkbox" className={classes.checkbox}>
-        <CheckBox
-          checked={itemSelectionStatus}
-          inputProps={{ 'aria-labelledby': labelIdentifier }}
-        />
+        {!data.injectionFailed && (
+          <CheckBox
+            checked={itemSelectionStatus}
+            inputProps={{ 'aria-labelledby': labelIdentifier }}
+          />
+        )}
       </StyledTableCell>
-      <StyledTableCell>
-        <Typography
-          className={classes.tableObjects}
-          style={{ maxWidth: '15rem' }}
-        >
+      <StyledTableCell className={classes.flexObject}>
+        {!data.injectionFailed && (
           <div
             className={classes.colorCircle}
             style={{ background: data.legendColor }}
           />
-          {data.chaosResultName}
-        </Typography>
-      </StyledTableCell>
-      <StyledTableCell>
-        <Typography
+        )}
+        <TextPopOver
+          text={data.chaosResultName}
           className={classes.tableObjects}
-          style={{ maxWidth: '12.5rem' }}
-        >
-          {data.workflow}
-        </Typography>
+          style={{ maxWidth: '15rem' }}
```

Comment:
please remove inline styles

Reply:
we wanted to override a single property so it has been used this way